hadoop-ozone/dist/src/main/compose/ozone-net-topology/network-config

```diff
@@ -0,0 +1,6 @@
+10.5.0.4	/rack1
+10.5.0.5	/rack1
+10.5.0.6	/rack1
+10.5.0.7	/rack2
+10.5.0.8	/rack2
```

Comment:
whitespace:tabs in line

Reply:
All Whitespade are intended to separate two columns.